lib/app/data/models/alarm_model.dart

```diff
@@ -1,6 +1,6 @@
 import 'dart:convert';
 
-import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:cloud_firestore/cloud_firestore.dart' as firestore;
```

Comment:
Any reason this was done? There's no name space conflicts here right?

Reply:
I added this as the Query parameter that store the volumes had conflicts with the cloud_firestore and isar db, as Query is defined in both.